website/docs/automation/controlplanes/ack/provision-resources.md

```diff
@@ -1,70 +1,93 @@
+Next, we will need to update the regional endpoint for DynamoDB within the Configmap used by the Kustomization to update the **Carts** deployment.
 
 ```file
-manifests/modules/automation/controlplanes/ack/rds/k8s/rds-dbgroup.yaml
+manifests/modules/automation/controlplanes/ack/dynamodb/dynamodb-ack-configmap.yaml
 ```
 
-Finally, we can create the configuration for the RDS database itself with a `DBInstance` resource:
+Next we want to populate the environment variable within the manifest. Run
 
-```file
-manifests/modules/automation/controlplanes/ack/rds/k8s/rds-instance.yaml
-```
+```bash
 
-Apply this configuration to the Amazon EKS cluster:
+$ envsubst < ~/environment/eks-workshop/modules/automation/controlplanes/ack/dynamodb/dynamodb-ack-configmap.yaml \
+  | tee ~/environment/eks-workshop/modules/automation/controlplanes/ack/dynamodb/dynamodb-ack-configmap.yaml \
+  > /dev/null
 
-```bash wait=30
-$ kubectl apply -k ~/environment/eks-workshop/modules/automation/controlplanes/ack/rds/k8s
-securitygroup.ec2.services.k8s.aws/rds-eks-workshop created
-dbinstance.rds.services.k8s.aws/rds-eks-workshop created
-dbsubnetgroup.rds.services.k8s.aws/rds-eks-workshop created
 ```
 
-The ACK controllers in the cluster will react to these new resources and provision the AWS infrastructure it has expressed. For example, we can use the AWS CLI to query the RDS database:
+:::info
+This process uses the ```envsubst``` utility to rewrite the environment variable into the manifest to reflect the 
+current region.
+:::
+
 
-```bash
-$ aws rds describe-db-instances \
-    --db-instance-identifier ${EKS_CLUSTER_NAME}-catalog-ack
+We're now finally able to apply these configurations to the Amazon EKS cluster:
+
+```bash wait=30
+$ kubectl apply -k ~/environment/eks-workshop/modules/automation/controlplanes/ack/dynamodb
+configmap/carts-ack created
+table.dynamodb.services.k8s.aws/items created
+deployment.apps/carts configured
 ```
+:::info
+Note that the above output only is describing the changed resources and not all. The final output you see may differ.
+:::
+
 
-It takes some time to provision the AWS managed services, in the case of RDS up to 10 minutes. The AWS provider controller will report the status of the reconciliation in the status field of the Kubernetes custom resources.
+The ACK controllers in the cluster will react to these new resources and provision the AWS infrastructure it has expressed. Using the AWS CLI, we can simply run
 
 ```bash
-$ kubectl get dbinstances.rds.services.k8s.aws ${EKS_CLUSTER_NAME}-catalog-ack -n catalog -o yaml | yq '.status'
+$ aws dynamodb list-tables
 ```
 
-We can use this status field to instruct `kubectl` to wait until the RDS database has been successfully created:
+Creating a DynamoDB table is almost instantaneous, so you should see the Items table created. If not, wait a few seconds and try again.
```

Comment:
I think its valuable to show using `kubectl wait` like we did before, especially helpful for tests that could beat the table getting created.